### PR TITLE
Refactor: Abstract notif component for auth pages.

### PR DESCRIPTION
**Pull request by tech4him1**
**- Summary**

The notification component is duplicated across some of the backends, and is missing on others. This PR moves the notif component above the auth components.

**- Test plan**

Tested GitHub backend: Error notifications show up correctly. 

**- Description for the changelog**

Abstract notif component for auth pages.

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/backends/github/AuthenticationPage.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ import React from 'react';
 import Button from 'react-toolbox/lib/button';
 import Authenticator from '../../lib/netlify-auth';
 import { Icon } from '../../components/UI';
-import { Notifs } from 'redux-notifications';
 import { Toast } from '../../components/UI/index';
```
Comment (erquhart) [inline src/backends/github/AuthenticationPage.js]: We can drop this from both pages now.

Comment (tech4him1) [reply src/backends/github/AuthenticationPage.js]: It is, right?

**Comment by tech4him1** [comment]
@erquhart Suggestions addressed and rebased.

**Comment by verythorough** [comment]
Deploy preview ready!

Built with commit 357aa8fb76b1cb13bd314cfd95e336b7c014f602

https://deploy-preview-747--netlify-cms-www.netlify.com

**Comment by tech4him1** [comment]
@erquhart This is rebased on 1.0, it should be good to merge.

**Comment by erquhart** [comment]
Restarted test builds, good to merge once they pass.

**Comment by verythorough** [comment]
Deploy preview ready!

Built with commit 5ed76b40ebc55f43ec06cdfd5fc2b97e5d2d739d

https://deploy-preview-747--cms-demo.netlify.com

**Pull request merged**